tools/pyorac/definitions.py

```diff
@@ -536,12 +536,62 @@ class ParticleType:
     wvl - Wavelengths used (negative implies the second view)
     sad - SAD file directory to use
     ls  - If true, process land and sea separately
+    mb  - 'm' for monochromatic calculation or 'b' for band calculation
+           weighted by the spectral response function
+    atmospheric_model - code denoting the atmospheric model where
+                         00: cloud, no Rayleigh scattering
+                         01: cloud, includes Rayleigh scattering
+                         1X: aerosol, where X denotes the gaseous model where
+                                1 = Tropical Atmosphere
+                                2 = Midlatitude Summer
+                                3 = Midlatitude Winter
+                                4 = Subarctic Summer
+                                5 = Subarctic Winter
+                                6 = 1976 US Standard
+    microphysical_model - Three digit code to represent particle microphysics.
+                          Backwards compatible with old LUT naming
+                          e.g. a70 for aerosol dust model
+    version - LUT version
     """
 
-    def __init__(self, name, inv=(), sad="CCI_A70-A79"):
+    def __init__(self, name, inv=(),
+                 sad="CCI_A70-A79",
+                 mb='m',
+                 atmospheric_model='01',
+                 microphysical_model='old',
+                 version='06'):
```

Comment:
We should set the default to `07` now that Don has updated to version 7.

Reply:
I thought only some tables moved to v7?